applications/user-generated-content.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Machine translation for user-generated content
+---
+
+# User-Generated Content
+
+Machine translation for **user-generated content** (UGC) is used mainly for **gisting**.
+
+
+### Goals
+Machine translation can increase key platform metrics like user engagement and sales.
+The amount of user-generated content published daily is non-stop. User-generated content is a good fit for machine translation because it is very high-scale.
```

Comment:
These 2 sentences can be combined.

*non-stop* is too slangy for this project.